script.js

```diff
@@ -1,4 +1,200 @@
+// generate random dice number
+function rollDice() {
+  let randomNum = Math.floor(Math.random() * 6) + 1;
+  return randomNum;
+}
+// arrange the order of dices as per user input.
+function arrangeDice(userInput, firstDice, secondDice) {
+  let arrangedResult;
+  if (userInput == 'dice1') {
+    diceOrder = 'dice1';
+  }
+  else {
+    diceOrder = 'dice2';
+  }
+  if (diceOrder == 'dice1') {
+    arrangedResult = `${firstDice}${secondDice}`;
+  }
+  else {
+    arrangedResult = `${secondDice}${firstDice}`;
+  }
+  return Number(arrangedResult);
+}
+// function is used for finding winner with highest (normal) as well as lowest score modes.
```

Comment:
What does function take as input and receive as output? Inputs and outputs should be clear from comment as well.